nbtc/relayer_test.go

```diff
@@ -40,6 +43,16 @@ var relayerConfig = RelayerConfig{
 	SignReqFetchLimit:  5,
 }
 
+func initTestDB(t *testing.T) dal.DB {
+	t.Helper()
+
+	db, err := dal.NewDB("file::memory:?cache=shared")
```

Comment:
Can you add comments about this problem? This should be noted in the source code.

Reply:
Added